classes/Link.php

```diff
@@ -750,7 +750,7 @@ public function getAdminLink($controller, $withToken = true, $sfRouteParams = ar
 
         $idLang = Context::getContext()->language->id;
 
-        return $this->getBaseLink().basename(_PS_ADMIN_DIR_).'/'.Dispatcher::getInstance()->createUrl($controller, $idLang, $params, false);
+        return '/'.basename(_PS_ADMIN_DIR_).'/'.Dispatcher::getInstance()->createUrl($controller, $idLang, $params, false);
```

Comment:
bad idea, this won't work with subdomains anymore imo

Reply:
hi, 
 ( after proposing same change with same mistake, sorry)
i tried : 
```php
return basename(__PS_BASE_URI__).'/'.basename(_PS_ADMIN_DIR_).'/'.Dispatcher::getInstance()->createUrl($controller, $idLang, $params, false);
```
but __PS_BASE_URI__ is still a data comming from shop_url 
if someone have a solution :)